server/controllers/loginUser.js

```diff
@@ -0,0 +1,40 @@
+const jwt = require('jsonwebtoken');
+const loginQuery = require('../database/queries/loginQuery');
+const { comparePassword } = require('../utils/hashPassword');
+const getHashPassword = require('../database/queries/getHashPassword');
+const { loginSchema } = require('../utils/validation/loginValidation');
+
+const checkLogin = (req, res) => {
+  loginSchema.validateAsync(req.body)
+    .then((validate) => {
+      loginQuery(validate.email)
+        .then((result) => {
+          if (result.rowCount) {
+            getHashPassword(validate.email)
+              .then((hashed) => {
+                comparePassword(validate.password, hashed.rows[0].pasword, (err, resu) => {
+                  if (resu) {
+                    jwt.sign({ is_Admin: true }, process.env.SECRET_KEY, (error, token) => {
```

Comment:
I expect your code to be more  formatted and take Lina's notes into consideration